templates/certified/model-details/_components-section.html

```diff
@@ -1,113 +1,84 @@
-          {% endif %}
-          {% else %}
-          <td data-heading="20.04 LTS"></td>
-          {% endif %}
-          {% if '22.04 LTS' in component.lts_releases %}
-          {% if component.lts_releases['22.04 LTS'][0].third_party_driver %}
-          <td data-heading="22.04 LTS">
-            {{ image(url="https://assets.ubuntu.com/v1/8dd0626d-3rd-party-icon.svg", alt="May require third-party driver", width="21", height="14", hi_def=True, attrs={"style":"margin-left: 0.45rem;"}, loading="lazy") | safe }}
-          </td>
-          {% elif component.lts_releases['22.04 LTS'][0].status == 'inprogress' %}
-          <td data-heading="22.04 ESM">
-            <i class="p-icon--help">In progress</i>
-          </td>
-          {% elif component.lts_releases['22.04 LTS'][0].status == 'certified' %}
-          <td data-heading="22.04 LTS">
-            <i class="p-icon--success">Supported</i>
-          </td>
-          {% endif %}
-          {% else %}
-          <td data-heading="22.04 LTS">
-            <i class="p-icon--error"></i>
-          </td>
-          {% endif %}
-          {% if '24.04 LTS' in component.lts_releases %}
-          {% if component.lts_releases['24.04 LTS'][0].third_party_driver %}
-          <td data-heading="24.04 LTS">
-            {{ image(url="https://assets.ubuntu.com/v1/8dd0626d-3rd-party-icon.svg", alt="May require third-party driver", width="21", height="14", hi_def=True, attrs={"style":"margin-left: 0.45rem;"}, loading="lazy") | safe }}
-          </td>
-          {% elif component.lts_releases['24.04 LTS'][0].status == 'inprogress' %}
-          <td data-heading="24.04 ESM">
-            <i class="p-icon--help">In progress</i>
-          </td>
-          {% elif component.lts_releases['24.04 LTS'][0].status == 'certified' %}
-          <td data-heading="24.04 LTS">
-            <i class="p-icon--success">Supported</i>
-          </td>
-          {% endif %}
-          {% else %}
-          <td data-heading="24.04 LTS">
-            <i class="p-icon--error"></i>
-          </td>
-          {% endif %}
-          <td data-heading="Comments">{{ component.note }}</td>
-        </tr>
-        {% endfor %}
-      </tbody>
-    </table>
-    <ul class="p-inline-list" style="margin-left:0.5rem;">
+    <div style="overflow-x: auto;">
+      <table class="p-table--mobile-car p-table-component-devices"
+             aria-label="Component Devices table">
+        <colgroup>
+          <col class="col-component" />
+          <col class="col-release" />
+          <col class="col-status" />
+          <col class="col-comments" />
+        </colgroup>
+
+        <thead>
+          <tr>
+            <th scope="col">Component</th>
+            <th scope="col">Ubuntu release</th>
+            <th scope="col">Status</th>
+            <th scope="col">Comments</th>
+          </tr>
+        </thead>
+
+        <tbody>
+          {% set releases = ["24.04 LTS", "22.04 LTS", "20.04 LTS", "CORE 20", "18.04 LTS", "CORE 18"] %}
```

Comment:
```suggestion
          {% set releases = ["24.04 LTS", "Core 24",  "22.04 LTS", "Core 22", "20.04 LTS", "Core 20", "18.04 LTS", "Core 18"] %}
```
Add more Ubuntu Core releases

Reply:
Another option for the ordering is 
`["24.04 LTS", "22.04 LTS", "20.04 LTS", "18.04 LTS", "Core 24",  "Core 22",  "Core 20", "Core 18"] `

@eliman11 Could you please advise whether it's clearer to list all LTS first before Core releases, or alternatively?